### ISSUE TITLE
node --use_strict fails silently

### ISSUE THREAD

**Issue by mLuby** [wallabyjs/public#302]
repro: use the [wallaby-node-iojs-sample](https://github.com/wallabyjs/wallaby-node-iojs-sample/) with the following changes:
1. remove `'use strict';` from calculator.js and calculatorSpec.js.
2. replace `--harmony --harmony_arrow_functions` with `--use_strict` in wallaby.js

After restarting wallaby, it reports 0 out of 0 tests (should be 1 passed); debugger shows this error:
```javascript
Sun, 25 Oct 2015 18:45:43 GMT wallaby:project File cache requires some updates, waiting required files from IDE
Sun, 25 Oct 2015 18:45:43 GMT wallaby:extended-core New document or complex document change
Sun, 25 Oct 2015 18:45:43 GMT wallaby:project File lib/calculator.js changes are patches only: false
Sun, 25 Oct 2015 18:45:43 GMT wallaby:project File test/calculatorSpec.js changes are patches only: false
Sun, 25 Oct 2015 18:45:43 GMT wallaby:project Preparing to process lib/calculator.js
Sun, 25 Oct 2015 18:45:43 GMT wallaby:project Starting processor pool (if not yet started)
Sun, 25 Oct 2015 18:45:43 GMT wallaby:project Preparing to process test/calculatorSpec.js
Sun, 25 Oct 2015 18:45:43 GMT wallaby:project Starting processor pool (if not yet started)
Sun, 25 Oct 2015 18:45:43 GMT wallaby:nodeRunner Error in worker: TypeError: Cannot read property 'addEventListener' of undefined
    at /Users/me/.atom/packages/atom-wallaby/wallaby/server.js:18:23847
    at Object.32 (/Users/me/.atom/packages/atom-wallaby/wallaby/server.js:19:3781)
    at e (/Users/me/.atom/packages/atom-wallaby/wallaby/server.js:14:254)
    at /Users/me/.atom/packages/atom-wallaby/wallaby/server.js:14:305
    at Object.n (/Users/me/.atom/packages/atom-wallaby/wallaby/server.js:16:21012)
    at b.exports (/Users/me/.atom/packages/atom-wallaby/wallaby/server.js:16:22734)
    at Object.39../lib/app (/Users/me/.atom/packages/atom-wallaby/wallaby/server.js:19:19316)
    at e (/Users/me/.atom/packages/atom-wallaby/wallaby/server.js:14:254)
    at a (/Users/me/.atom/packages/atom-wallaby/wallaby/server.js:14:421)
    at Object.<anonymous> (/Users/me/.atom/packages/atom-wallaby/wallaby/server.js:14:438)
```

**Comment by ArtemGovorov** [comment]
Thanks for the repro steps, the issue is fixed in core v1.0.134.

**Comment by mLuby** [comment]
Thanks! :+1:
